TheMET/TheMET/ViewControllers/ArtistsSectionViewController.swift

```diff
@@ -41,23 +41,26 @@ class ArtistsSectionViewController: UIViewController {
             self.featuredArtistsView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor)
         ])
         self.featuredArtistsView.onViewTapped = { [weak self] in
-            self?.featuredArtistsViewDidTap()
+            self?.viewModel?.featuredArtistsViewDidTap()
         }
-        self.artistsServiceSubscriber = self.artistService.$artistsList
-            .sink(receiveValue: { [weak self] newArtistsList in
-                let count = newArtistsList.count
-                self?.featuredArtistsView.title = self?.featuredArtistsViewTitle(artistsCount: count)
-            })
-    }
-    
-    private func featuredArtistsViewDidTap() {
-        let artistsViewController = ArtistsCatalogViewController()
-        self.navigationController?.pushViewController(artistsViewController, animated: true)
+        self.setUpViewModel()
     }
     
-    private func featuredArtistsViewTitle(artistsCount: Int) -> String {
-        let formatString: String = NSLocalizedString("artists count", comment: "")
-        return String.localizedStringWithFormat(formatString, artistsCount)
+    private func setUpViewModel() {
+        let viewModel = ArtistsSectionViewModel(presentingControllerProvider: { [weak self] in
+            return self
+        })
+        self.viewModel = viewModel
+        self.titleSubscriber = viewModel.$title.sink(receiveValue: { newTitle in
+            self.featuredArtistsLabel.apply(font: NSLocalizedString("serif_font", comment: ""), color: UIColor(named: "pear"), fontSize: 16, title: newTitle)
+        })
+        self.imageSubscriber = viewModel.$image.sink(receiveValue: { newimage in
+            guard let image = newimage else {return}
```

Comment:
The `View` must show exactly what its `ViewModel` says. If `newimage` is `nil` then `View` must show no image.
Alternatively, make `ViewModel.image` non-optional, so `ViewModel` couldn't provide `nil` image.